packages/ag-charts-community/src/module/optionsModule.ts

```diff
@@ -332,7 +332,19 @@ export class ChartOptions<T extends AgChartOptions = AgChartOptions> {
             return [{}, annotationsThemes];
         }
 
-        return [{ annotations: { axesButtons, enabled, optionsToolbar, toolbar } }, annotationsThemes];
+        const { presetType } = this.optionMetadata;
+
+        if (presetType == 'price-volume') {
```

Comment:
Can we make this less FC specific, and assume that for preset cases they are passing the right options?

Practically:
```suggestion
        if (presetType != null) {
```

Reply:
Discussed and updated - we don't need to be this specific here.